panoptes_cli/commands/subject_set.py

```diff
@@ -140,6 +217,12 @@ def modify(subject_set_id, display_name):
     help=("Do not abort when creating subjects with no media files."),
     is_flag=True,
 )
+@click.option(
+    '--compress',
+    '-C',
+    help=("Compress image files if larger than file size."),
```

Comment:
Specify "limit":
```suggestion
    help=("Compress image files if larger than file size limit."),
```

Reply:
Also added file size estimate to the help text